resp3_replies.json

```diff
@@ -981,24 +987,24 @@
     "* [Map reply](../topics/protocol.md#maps): List of configurable time parameters and their values (milliseconds)."
   ],
   "SENTINEL FAILOVER": [
-    "[Simple string reply](../topics/protocol.md#simple-strings): `OK`. Force a fail over as if the master was not reachable, and without asking for agreement to other Sentinels."
+    "[Simple string reply](../topics/protocol.md#simple-strings): `OK`. Force a fail over as if the primary was not reachable, and without asking for agreement to other Sentinels."
   ],
   "SENTINEL FLUSHCONFIG": [
     "[Simple string reply](../topics/protocol.md#simple-strings): `OK`. Force Sentinel to rewrite its configuration on disk, including the current Sentinel state."
   ],
-  "SENTINEL GET MASTER-ADDR-BY-NAME": [],
+  "SENTINEL GET PRIMARY-ADDR-BY-NAME": [],
   "SENTINEL HELP": [
     "[Array reply](../topics/protocol.md#arrays): Helpful text about subcommands."
   ],
   "SENTINEL INFO CACHE": [
-    "[Array reply](../topics/protocol.md#arrays): This is actually a map, the odd entries are a master name, and the even entries are the last cached INFO output from that master and all its replicas."
+    "[Array reply](../topics/protocol.md#arrays): This is actually a map, the odd entries are a primary name, and the even entries are the last cached INFO output from that primary and all its replicas."
   ],
-  "SENTINEL IS MASTER-DOWN-BY-ADDR": [],
-  "SENTINEL MASTER": [
-    "[Map reply](../topics/protocol.md#maps): The state and info of the specified master."
+  "SENTINEL IS PRIMARY-DOWN-BY-ADDR": [],
```

Comment:
Both names exist for backward compatibility. We should probably duplicate these for both the primary and master subcommands.

Reply:
I just did "robotic" replacements here, but actually:

1. The sentinel commands appear only in the resp3 file
2. The sentinel command documentation lives outside of the regular command documentation anyway: https://valkey.io/topics/sentinel/#sentinel-api

I tend to remove the Sentinel commands from here. But I can also duplicate these both for master/primary and resp2/resp3 If we intend to make the sentinel commands "first class citizens" in the future.